src/Tests/RedirectUITest.php

```diff
@@ -166,13 +166,33 @@ public function testRedirectUI() {
     ), t('Save'));
     $this->assertRaw(t('The anchor fragments are not allowed.'));
 
+    // Test filters.
+    // Filter  with other value.
+    $this->drupalGet('admin/config/search/redirect', array('query' => array(
+      'status_code' => '401',
```

Comment:
use a 3xx status code, only those are valid values and we'll eventually limit it to that.